pagopa/io-app
# fix(IT Wallet): [SIW-1873] Fix `useMaxBrightness` flickering on iOS

## Short description
On iOS, displaying the QR Code of the Trustmark causes the screen brightness to flicker. This issue is due to a double animation triggered by an unexpected `AppState` change.
This PR introduces a check to prevent the double animation, ensuring smooth screen brightness transitions without flickering.

## List of changes proposed in this pull request
- Added a check on app state change in `useMaxBrigthness`

## How to test
Open the Trustmark QR Code modal on iOS, check that the brightness is not flickering and reaches the maximum value smoothly
